ast/tests/Earthfile

```diff
@@ -1,4 +1,4 @@
-VERSION --pass-args 0.6
+VERSION --pass-args 0.7
 FROM alpine:3.15
```

Comment:
since I saw you updated it above
```suggestion
FROM alpine:3.18
```

Reply:
`find . -type f | xargs -n 1 sed -i 's/alpine:3.[0-9]\+/alpine:3.18/g'` :nerd_face: